views/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    
+  </head>
+  <body>
+    test test test
+  </body>
+</html>
```

Comment:
Is this our main html file? Is views inside public or is it the same as public? Just to make sure I understand the structure :wink:

Reply:
views would be the equivalent of our public folder. It appears that when setting up this scaffold Matt used the original file structure/naming. This is something we can change/adapt to fit the file structure we've discussed previously.